server/views/pages/appointments/create/check-answers.njk

```diff
@@ -11,7 +11,7 @@
 
 {% set summaryListRows = [] %}
 
-{% if session.createAppointmentJourney.type == 'individual' %}
+{% if session.createAppointmentJourney.type == 'INDIVIDUAL' %}
```

Comment:
I'd add these as globals in `nunjucksSetup.ts` like with the `YesNo` and `AppointmentRepeatPeriod` enums so you can use those rather than strings